### PR TITLE
Encryption

### PR DESCRIPTION
**Pull request by cclevinger**
Added encryption for student names.

### CODE REVIEW

File: decrypt_test.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  print(p_name)
+'''
+
+print("Team 0:")
+print(students.get_students(0, 0))
+
+students.insert_student("Dame Dolla", 1990, 0, "Team 0")
+print("Team 0:")
+print(students.get_students(0, 0))
+
```
Comment (J8cob1) [inline decrypt_test.py:25]: Some more comments here might be nice?

**Review by J8cob1 — CHANGES_REQUESTED** [review]
When I reviewed your changes, I made comments on the previous pull request (that is now closed). I think most of what I wrote is still valid for this one though:

Is your task complete? - The goal is "Student names are encrypted in the database and decrypted leaving the database." I think we might want to have some discussion about this. I am not sure it is.

Is the branch caught up to master/are there merge conflicts still to be resolved? - It is caught up to master.

Are old files unnecessarily introduced? - key.txt

Is old code unnecessarily overwritten? - I don't think so?

Are new issues introduced to existing code? Do existing automated tests continue to work?
-Unit Tests: one of them is broken (though this is already known)
-Log in: seems to work fine
-Dashboard/Edit: broken
-Review: broken

Flake8 (the version I am using, at least):
capstone360/setup_mockup_db.py:116:41: E231 missing whitespace after ','
decrypt_test.py:1:1: F401 'os' imported but unused
decrypt_test.py:25:1: W391 blank line at end of file
gbmodel/model_sqlalchemy.py:123:60: E261 at least two spaces before inline comment
gbmodel/model_sqlalchemy.py:135:27: E261 at least two spaces before inline comment

Note 1: I noticed that the GitHub line numbers don't match the flake8 ones (135 in flake8 was 136 in GitHub). So you are aware.

Note 2: I tried to exclude flake8 output that wasn't related to the changes made here (we have other problems)

uses 4 spaces where appropriate, not tabs? - It looks like it

camelCase for class names, snake_case for variable and function names? - I skimmed it a couple of times and didn't notice any problems

code is commented? - mostly. Might want some in his test

Does the code handle edge cases? Are try/except statements used sufficiently? - nope, though exception handling could take place outside of the gbmodel/model_sqlalchemy calls as well

Is there documentation on how to test your code somewhere? - no
